bcmc/src/main/java/com/adyen/checkout/bcmc/BcmcComponentProvider.kt

```diff
@@ -49,12 +49,13 @@ import com.adyen.checkout.sessions.repository.SessionRepository
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class BcmcComponentProvider(
-    overrideComponentParams: ComponentParams? = null,
+    private val overrideComponentParams: ComponentParams? = null,
+    private val checkoutSession: CheckoutSession? = null
```

Comment:
It's better to give this another name to avoid confusion in the `get()` where we pass the same thing. Maybe `dropInCheckoutSession` or `overrideCheckoutSession` is a good name 🤔